lettuce/cli.py

```diff
@@ -119,7 +127,12 @@ def convergence(ctx):
             resolution, error_u, factor_u/2, error_p, factor_p/2))
     return 0
 
+def flow_by_namee():
```

Comment:
Doubly defined (see `flows/__init__.py`).

Just `from lettuce.flows import flow_by_name` should work